innsending/src/test/kotlin/no.nav.helsearbeidsgiver.inntektsmelding.innsending/KvitteringServiceTest.kt

```diff
@@ -34,16 +34,15 @@ import java.util.UUID
 class KvitteringServiceTest :
     FunSpec({
         val testRapid = TestRapid()
-        val mockRedis = MockRedis(RedisPrefix.Kvittering)
+        val mockRedisStore = mockk<RedisStore>(relaxed = true)
```

Comment:
`MockRedis` etterligner redis med lagring og lesing av midlertidige verdier. Siden vi ikke trenger det for stateless, så kan vi bruke en enklere `mockk<RedisStore>`. Dette gjelder også de andre testene som har den samme endringen.